relayer/chains/substrate/key.go

```
@@ -0,0 +1,125 @@
+package substrate
+
+import (
+	"os"
+
+	"github.com/cosmos/go-bip39"
+	"github.com/cosmos/relayer/v2/relayer/chains/substrate/keystore"
+	"github.com/cosmos/relayer/v2/relayer/provider"
+)
+
+func (sp *SubstrateProvider) CreateKeystore(path string) error {
+	keybase, err := keystore.New(sp.PCfg.ChainID, sp.PCfg.KeyringBackend, sp.PCfg.KeyDirectory, sp.Input)
+	if err != nil {
+		return err
+	}
+	sp.Keybase = keybase
+	return nil
+}
+
+func (sp *SubstrateProvider) KeystoreCreated(path string) bool {
+	if _, err := os.Stat(path); err != nil || sp.Keybase == nil {
+		return false
```

why not return the error? might be used for debugging